devel-common/src/tests_common/test_utils/db.py

```diff
@@ -335,7 +335,9 @@ def clear_dag_specific_permissions():
 
 def clear_all():
     clear_db_runs()
+    clear_db_backfills()
     clear_db_assets()
+    clear_db_triggers()
```

Comment:
Is also cleared as part of `clear_db_runs` but association table is not, so for safety I added it too.